Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_008.AwsCredentials.sarif

```diff
@@ -74,6 +74,7 @@
             }
           ],
           "fingerprints": {
+            "AssetFingerprint/v1": "[id=AKIAIOSFODNN7EXAMPLE]",
```

Comment:
>AssetFingerprint [](start = 13, length = 16)

We are missing an AWS platform here. I'll stop making this comment for other platforms. :)